app/cdap/components/PipelineHistory/styles.ts

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import styled from 'styled-components';
+
+export const PipelineHistoryTableDiv = styled.div`
+  &.pipeline-history-list-table.grid-wrapper {
+    background-color: white;
+    padding: 0 4px;
+    height: calc(100% - #{$header-height});
+
+    .grid.grid-container {
+      max-height: 100%;
+
+      .grid-header > .grid-row {
+        border-width: 3px;
+
+        .sortable {
+          cursor: pointer;
+        }
+
+        .fa.fa-lg {
```

Comment:
is this font awesome? also you couldn't break out these styled components so you wouldn't have to use classNames in the code above?